addons/web/static/tests/core/popover/popover_tests.js

```diff
@@ -158,3 +158,72 @@ QUnit.test("reposition popover should properly change classNames", async (assert
     );
     assert.strictEqual(arrow.className, "popover-arrow top-auto");
 });
+
+QUnit.test("arrow height is taken into account when positioning popover", async (assert) => {
+    // Force some style, to make this test independent of screen size
+    const container = document.createElement("div");
+    container.id = "container";
+    container.style.backgroundColor = "pink";
+    container.style.height = "450px";
+    container.style.width = "450px";
+    container.style.display = "flex";
+    container.style.alignItems = "center";
+    container.style.justifyContent = "center";
```

Comment:
```suggestion
    container.style = {
        ...container.style,
        backgroundColor: "pink",
        etc.
    };
```
Isn't that possible? :thinking:

Reply:
Also, is the background color necessary?

Reply:
> Also, is the background color necessary?

Isn't it Barbie's month :unicorn: :sparkles:  ? 
Actually I've copy paste the style from the previous test which was made to avoid repositioning and it was working well so I did not change it afterward. I'll remove what can be removed :slightly_smiling_face: (I think it was initially used to be able to identify blocks in test debug mode)